docs/working-with-projects/working-with-images/pulling-pushing-images.md

```diff
@@ -73,28 +73,28 @@ Users can click the **Registry Certificate** button to download the registry cer
 
 ## Deleting Repositories
 
-Deleting repositories involves two steps.
+Deleting a repository involves two steps.
 
-First, you delete a repository in the Harbor interface. This is soft deletion. You can delete the entire repository or just one of its tags. After the soft deletion, the repository is no longer managed by Harbor, however, the repository files remain in the Harbor storage.
+First, you delete the repository in the Harbor interface. You can delete the entire repository or just one of its tags. This is soft deletion. After the soft deletion, the repository is no longer managed by Harbor, but the repository files remain in the Harbor storage.
 
 ![browse project](../../../img/new-delete-repo.png)
 ![browse project](../../../img/new-delete-tag.png)
 
 {{< danger >}}
-If both tag A and tag B refer to the same image, after deleting tag A, B will also get deleted. if you enabled content trust, you need to use notary command line tool to delete the tag's signature before you delete an image.
+If multiple tags refer to the same image, then deleting one tag results in the deletion of all tags for that image. For example, if both tag A and tag B refer to the same image, then when you delete tag A, tag B is also deleted. If you enabled content trust, you need to use the notary command-line tool to delete the tag's signature before you delete an image.
```

Comment:
this is no longer true. I will rework this section after we merge this PR.  Please ignore this section for now @LandoVMW

Reply:
Yes, this is no longer true.

Reply:
Thanks. warning deleted.